### ISSUE TITLE
Broken `OktaAuth.isAuthenticated` not aware of `tokenManager` config in options because they are not passed in correctly

### ISSUE THREAD

**Issue by theseyi** [okta/okta-auth-js#924]
In the latest version of OktaAuth, isAuthenticated has the following comment 
```js
  // Returns true if both accessToken and idToken are not expired
  // If `autoRenew` option is set, will attempt to renew expired tokens before returning.
```
The implementation correctly looks up the values for `autoRenew` and `autoRemove`
https://github.com/okta/okta-auth-js/blob/859798bf565a8c1a7d4459c8fb57f691bd3c6b91/lib/OktaAuth.ts#L503-L512

However, these values are always going to be `undefined` because the `tokenManager` configuration is not correctly passed in! The reason is because `buildOptions` (see below) does not copy over the values for `tokenManager` from the options passed to the OktaAuth instance
```
const options = {
  clientId: CLIENT_ID,
  issuer: ISSUER,
  redirectUri: LOGIN_REDIRECT,
  scopes: ['openid', 'profile', 'email', 'offline_access'],
  postLogoutRedirectUri: POST_LOGOUT_REDIRECT,
  pkce: true,
  tokenManager: {
    autoRenew: true,
    autoRemove: true,
  },
};

const authClient = new OktaAuth(oidc)
```
#### Screenshot of result from above statements
![image](https://user-images.githubusercontent.com/3725821/131789890-53598089-0b5a-4179-a159-845ccea58ba3.png)

https://github.com/okta/okta-auth-js/blob/c5ae1eb0869f10f42ad8df0591db2b45c2e27c40/lib/options.ts#L117-L155

In the call to removeNils here there is not key for `tokenManager` so the resulting object will not have this value

**Comment by denysoblohin-okta** [comment]
Thanks for submitting this issue and find the cause in the code!
Internal ref: OKTA-425705

**Comment by shuowu** [comment]
Fixed in [5.4.3](https://github.com/okta/okta-auth-js/releases/tag/okta-auth-js-5.4.3)
